src/main/java/life/qbic/projectbrowser/controllers/DataHandler.java

```diff
@@ -2669,6 +2669,48 @@ public String getSecondaryName(Sample samp, String datsetSecName) {
     return secondaryName.replace("__", "_").replaceAll("^_+", "").replaceAll("_+$", "");
   }
 
+  /**
+   * generates informative description from dataset secondary name and the dataset's experiment,
+   * given its ID. should only be used if a dataset does not have an associated sample
+   * 
+   * @param datasetSecondaryName
+   * @param experimentID
+   * @return
+   */
+  public String generateOrphanDatasetInformation(String datasetSecondaryName, String experimentID) {
```

Comment:
It is not really orphan right?

Reply:
true, maybe "composeDatasetWithoutSampleDescription"?

Reply:
`retrieveDatasetInfoWithoutSample` ?

Reply:
i like it